src/vivarium/framework/population/population_view.py

```diff
@@ -4,11 +4,11 @@
 ===================
 
 The :class:`PopulationView` is a user-facing abstraction that manages read and write access
-to the underlying simulation :term:`State Table`. It has two primary responsibilities:
+to the underlying simulation :term:`state table`. It has two primary responsibilities:
 
     1. To provide user access to subsets of the simulation state table
        when it is safe to do so.
-    2. To allow the user to update the simulation state in a controlled way.
+    2. To allow the user to update private data in a controlled way.
```

Comment:
Note for the future: we should add "private column" and "public column/attribute" or whatever we end up calling these as terms to the vivarium glossary.

Reply:
I don't remember if this was part of the reason we settled on the term "attribute" throughout this effort, but that's actually in the glossary and defined exactly like we've been using it. So that's cool